src/lib/viewers/iframe/__tests__/IFrameLoader-test.js

```diff
@@ -18,4 +31,24 @@ describe('lib/viewers/iframe/IFrameLoader', () => {
             EXT: ['boxnote', 'boxdicom'],
         });
     });
+
+    test.each([
+        // disableDicom, fileType, viewerInstance
```

Comment:
In the future, consider using the [tagged template literal syntax](https://github.com/box/box-content-preview/pull/1436#discussion_r751671279) to incorporate these variable names directly into the inputs.

Reply:
Ohh, my bad I didn't know that actual syntax. Will update.